app/javascript/packs/components/DropdownOption.jsx

```diff
@@ -8,18 +8,28 @@ class DropdownOption extends React.Component {
         this.props.onSelect(selected_option_id, this.props.answer.text);
       }
     }
+
+    if (this.props.setFocusFunc) {
+      this.props.setFocusFunc(() => this.ref.focus());
+    }
   }
 
   handleChange(e) {
     const optionId = e.target.value;
     this.props.onSelect(optionId, this.props.options[optionId].text);
+    e.targer.blur();
```

Comment:
targer? 😂

Reply:
oh so THAT's why i get that `undefined` error, thank you